tests/integration/test_agentd/test_agentd_parametrized_reconnections.py

```diff
@@ -103,7 +103,10 @@
 remoted_server = None
 
 
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture()
```
Function should be the default scope.